Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:18.04
```

Comment:
probably just use `oasislabs/rust` so that the versions stay in sync. You won't need the apt stuff, either.